content/data_science/3_introduction_to_statistics/data_and_data_types_questions/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+_db_id: 108
+content_type: project
+flavours:
+- none
+prerequisites:
+  hard:
+  - content/data_science/3_introduction_to_statistics/data_and_data_types
+  soft: []
+protect_main_branch: false
+ready: true
+submission_type: repo
+title: Data and Data Types - Questions
+---
+
+Please answer the following questions:
+
+1. Out of the four data types that we have, What data type does the count of emails received each day represent?
```

Comment:
This sentence needs to be better. 

Int is a data type. So is float and string. These people know a little Python by now.

Please make all questions crystal clear.  If there is a 50% chance that learners will misunderstand the question then 50% of learners will misunderstand it. That means 50% of learners will need to be corrected by a human who gets paid by the hour to repeat themselves. 

Always think of the learner. Keep them at the center.

Please also make sure that all content has good grammer, spelling, punctuation, capitalisation and all the good things. We are delivering this content to professionals.  "What" should not have a capital letter. It is in the middle of a sentence.